### PR TITLE
Firebase deploys

### PR DESCRIPTION
**Pull request by ianmah**

### CODE REVIEW

File: .firebaserc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (kevin-zou) [inline .firebaserc:1]: Do we need this file if we're being explicit about which firebase project we're on in GitHub Actions? Or is this mostly just for fallback?

File: .github/workflows/firebase-hosting-merge.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+'on':
+  push:
+    branches:
+      - master
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm ci && npm run build
```
Comment (kevin-zou) [inline .github/workflows/firebase-hosting-merge.yml:14]: Do we also need to include the environment variables here like we did in the PR workflow?

**Review by kevin-zou — COMMENTED** [review]
Couple of comments but otherwise it's 🔥 Also wondering if we should regenerate all our secrets sometime soon. The prod (I think?) repo secrets were updated Aug 2020, might be good practice to refresh them yearly.

File: .firebaserc (hunk 1 above)
Comment (kozr) [reply .firebaserc:1]: this prob came with `firebase init`

Comment (kozr) [reply .firebaserc:1]: may be useful if we need to use firebase-cli

**Review by kozr — APPROVED** [review]
if all goes well then lgtm

File: .github/workflows/firebase-hosting-merge.yml (hunk 2 above)
Comment (ianmah) [reply .github/workflows/firebase-hosting-merge.yml:14]: We'll find out, but I don't think that was the issue

**Pull request merged**